File: cpp/4_158A_NextRound.cpp
#include<bits/stdc++.h>

int main(){
    int n, k, temp, ans = 0;
    std::vector<int> scores;
    std::cin>>n>>k;
    for (int i = 0; i < n; i++){
        std::cin>> temp;
        scores.push_back(temp);
    }
    temp = scores[k-1];
    for(auto i : scores){
        if (i >= temp && i > 0){
            ans++;
        }
    }
    std::cout<<ans;
    return 0;
}

File: python/4_158A_NextRound.py
def main():
    n, k = input().split()
    ans = 0
    scores = []
    n = int(n)
    k = int(k)
    scores = [int(el) for el in input().split()]
    check = scores[k-1]
    for i in scores:
        if i >= check and i > 0:
            ans += 1
    print(ans)

if __name__ == "__main__":
    main()